app/test_split_integer.py

```diff
@@ -1,21 +1,65 @@
+import unittest
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+class TestSplit(unittest.TestCase):
 
+    def test_split_integer_zero_parts(self) -> None:
```

Comment:
no need to delete original tests - you must use them as a boiler plate and add your code to the implementation.
Here you overcomplicated simple tests that should be one line.
No need to declare variables that will be used only once.
No need to write one-liner docstrings for tests - in this task optionally you can put an error message after assert to explain what went wrong e.g:
` assert split_integer(8, 1) == [8], value must be divided into a list with 1 element equal to value `